.github/workflows/new_contributor_pr.yml

```diff
@@ -0,0 +1,24 @@
+name: New Contributor Message
+
+on:
+  pull_request_target:
+
+jobs:
+  build:
+    name: Hello new contributor
+    runs-on: ubuntu-latest
+    steps:
+      # Pinned to v2.0
+      - uses: deborah-digges/new-pull-request-comment-action@224c179a9e23f65ec50ff3240b8716369dc415d7
```

Comment:
This action limits to first contribution yes? 🙇‍♂️🏃

Reply:
Yep - it lists PRs within the repository by that user: https://github.com/Deborah-Digges/new-pull-request-comment-action/blob/main/src/index.js#L9-L15.

It's probably simple enough to inline with a script [like I did here](https://github.com/django/django/pull/13961/files#diff-721c8b3a3c86e303692c0fd6f744bb92e76eaf44146145c9baa8b65d7099b9d4R33), but for now 🤷